src/jshint.js

```diff
@@ -1490,7 +1488,19 @@ var JSHINT = (function() {
   }
 
   function parseFinalSemicolon() {
+    if (lex.inContext(Context.Template)) return;
```

Comment:
I'm not sure if this is really needed... technically template expressions aren't parsed using statement() so they shouldn't need this...